dynawo/sources/Models/CPP/Events/ModelAreaShedding/DYNModelAreaShedding.cpp

```diff
@@ -277,10 +283,17 @@ ModelAreaShedding::defineParameters(vector<ParameterModeler>& parameters) {
   parameters.push_back(ParameterModeler("deltaTime", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));
   parameters.push_back(ParameterModeler("deltaP", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER, "*", "nbLoads"));
   parameters.push_back(ParameterModeler("deltaQ", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER, "*", "nbLoads"));
+  parameters.push_back(ParameterModeler("PShed", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER, "*", "nbLoads"));
```

Comment:
This is the line to initialize a table of parameters. Should be parameters.push_back(ParameterModeler("PShed", VAR_TYPE_DOUBLE, EXTERNAL_PARAMETER));